com.unity.formats.alembic/Documentation~/animClip.md

```diff
@@ -3,12 +3,16 @@
 You can control the playback of Alembic using Unity's Animation System bound to the GameObject that contains the [Alembic Stream Player](ref_StreamPlayer.md) component.
 
 1. Select the GameObject with the Alembic Stream Player component and open the [Animation window](https://docs.unity3d.com/Manual/animeditor-UsingAnimationEditor.html).
-2. [Create an Animator component and an Animation clip](https://docs.unity3d.com/Manual/animeditor-CreatingANewAnimationClip.html) if you don't already have them.
-3. [Save keyframes](https://docs.unity3d.com/Manual/animeditor-AnimatingAGameObject.html) for the **Time** property on the [**Alembic Stream Player**](ref_StreamPlayer.md) component.
+
+1. [Create an Animator component and an Animation clip](https://docs.unity3d.com/Manual/animeditor-CreatingANewAnimationClip.html) if you don't already have them.
```

Comment:
A little weird that every item now is suffixed "1.". 
But the rendered version is good, though.

Reply:
Actually, this is the way we should always identify a numbered line in markdown. It's easier to maintain the source (insertion, removal, reordering) and the rendered version takes care of the actual numbering.